main/abuild/0001-abuild-properly-detect-package-size-on-btrfs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/abuild.in b/abuild.in
+index 14955357af..38d2598b8f 100644
+--- a/abuild.in
++++ b/abuild.in
+@@ -812,7 +812,7 @@ prepare_metafiles() {
+ 	cd "$dir"
+ 	mkdir -p "$controldir"
+ 	local builddate=$(date -u "+%s")
+-	local size=$(du -sk | awk '{print $1 * 1024}')
++	local size=$(sync;du -sk | awk '{print $1 * 1024}')
```

Comment:
@fabled I understand your point it will be an impact. Currently this is so far the one way to fix it, you have to sync all the changes to disk before running ```du```.

I will investigate and hopefully find a way to make it conditional for BTRFS only.